bundesliga_app/views.py

```diff
@@ -7,7 +7,7 @@
 from django.shortcuts import redirect
 from django.shortcuts import render
 from django.utils.decorators import method_decorator
-from django.http import HttpResponseRedirect, HttpResponse
+from django.http import HttpResponseRedirect, HttpResponse, JsonResponse
```

Comment:
In order to keep imports ordered you can do:
`
from django.http import (
    HttpResponseRedirect,
    HttpResponse,
    JsonResponse
)
`

Reply:
yes, is a good advise... in fact I think there are some import that we are not using any more...